### ISSUE TITLE
Service param not being set in plugins that can use it in db-less mode

### ISSUE THREAD

**Issue by invisionmcole** [Kong/kubernetes-ingress-controller#361]
#174 # Summary

SUMMARY_GOES_HERE

The service https://docs.konghq.com/1.2.x/admin-api/#service-object is not passed into the plugins in db-less mode. 

**Kong Ingress controller version**

0.5.0

**Kong or Kong Enterprise version**

1.2.2

### Kubernetes version

17:58 $ kubectl version
Client Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.0", GitCommit:"fff5156092b56e6bd60fff75aad4dc9de6b6ef37", GitTreeState:"clean", BuildDate:"2017-03-28T16:36:33Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.8", GitCommit:"0c6d31a99f81476dfc9871ba3cf3f597bec29b58", GitTreeState:"clean", BuildDate:"2019-07-08T08:38:54Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}

### Environment

MiniKube

### What happened

Some plugins like https://docs.konghq.com/hub/kong-inc/response-transformer or https://docs.konghq.com/hub/kong-inc/zipkin/  use the {service} tag when db-less to associate the route with a plugin. When using db-less mode with the ingress controller this is nill. This causes the tracing collectors to tag them as unknown service.

example plugin output
```
{"next":null,"data":[{"created_at":1565904409,"config":{"sample_ratio":1,"http_endpoint":"http:\/\/zipkin.istio-system:9411\/api\/v2\/spans"},"id":"74bd9968-cbd1-5d14-9f77-15018c57a6c9","service":null,"name":"zipkin","tags":null,"consumer":null,"run_on":"all","protocols":["http","https"],"route":{"id":"97844f6a-2722-5856-bd46-7603a9cc5f3e"},"enabled":true}]}
```

### Expected behavior

Either the service to be defined in the metadata tag or the k8s service address.

### Steps To Reproduce

1. Setup the kong resources in DB-less mode via https://github.com/Kong/kubernetes-ingress-controller/blob/master/docs/deployment/minikube.md
2. Create an echo service via https://github.com/Kong/kubernetes-ingress-controller/blob/master/docs/guides/getting-started.md
3.  Apply the following yamls 

```
apiVersion: configuration.konghq.com/v1
kind: KongPlugin
metadata:
  name: zipkin
config:
  http_endpoint: http://zipkin.istio-system:9411/api/v2/spans
  sample_ratio: 1
plugin: zipkin
```

```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: demo-example-com
  annotations:
    plugins.konghq.com: zipkin
spec:
  rules:
  - http:
      paths:
      - path: /bar
        backend:
          serviceName: echo
          servicePort: 80
```

4.  curl -s $PROXY_IP/bar and see the tracing headers added but tagged as unknown service in Jaeger. 
![image](https://user-images.githubusercontent.com/39272119/63130258-d1088f00-bf87-11e9-8309-36f5260dcefc.png)

**Comment by hbagdi** [comment]
On an cursory look, It seems like this is another instance of https://github.com/Kong/kong-plugin-zipkin/issues/39.

**Comment by hbagdi** [comment]
This is solved with Kong 1.3. See above issue.
